README.md

```diff
@@ -1,3 +1,93 @@
+
+## Contributing
+
+Genkit is an open source framework, and we welcome contributions. Information on how to get started can be found in our [contributor guide](CONTRIBUTING.md).
+
+## Key features
+
+Gen AI models
+
+- **Unified API for generation** across AI models built by Google (Gemini, Gemma) and third party providers. Supports multimodal input, multimedia content generation, and custom options.
+- **Generate structured output** as strongly-typed objects with custom schemas for easy integration into your app.
+- **Define custom tools for your AI models** to fetch data, display UI, write to a database, and more.
+- **Write and manage prompts** with all relevant metadata encapsulated in a single dotprompt file. Supports handlebars templating, history, multimedia, and more.
+
+![Screenshot of IDE showing Firebase Genkit RAG sample code](docs/resources/readme-rag-screenshot.png)
+
+### Retrieval
+
+**Build context-aware AI features** by indexing your data and dynamically retrieving relevant information from your database. Genkit provides flexible, light-weight abstractions for indexers and retrievers that work with any database provider.
+
+### Evaluation
+
+Evaluate your end-to-end AI workflow using a variety of pre-built and custom evaluators. As easy as:
+
+```
+genkit eval:flow myAiWorkflow --input testQuestions.json
+```
+
+### Extensibility with plugins
+
+**Access pre-built components and integrations** for models, vector stores, tools, evaluators, observability, and more through Genkit’s open ecosystem of plugins built by Google and the community. To see this in action, you can explore Node.js plugins on [NPM](https://www.npmjs.com/search?q=keywords:genkit-plugin).
+
+You can also use this extensibility to easily define custom components whenever existing plugins don’t fit your needs.
+
+For more information:
+
+- Genkit plugins on NPM
+- Writing Genkit plugins
+
+### Deployment
+
+**Deploy your AI feature with a single command** through the Firebase or Google Cloud CLI to:
+
+- Cloud Functions for Firebase (Node.js only)
+- Google Cloud Run (Node.js or Go)
+
+You can also deploy to any container platform where your chosen runtime is supported.
+
+### Observability and monitoring
+
+Genkit is **fully instrumented with OpenTelemetry** and provides hooks to export telemetry data. Easily log traces and telemetry to Google Cloud using pre-built plugins or set up with a custom provider for full end-to-end observability and monitoring in production.
+
+## Genkit Developer UI
+
+Genkit’s developer UI is a local web app downloaded by the Genkit CLI tooling package that gives developers sandboxed runners the Genkit actions (models, prompts, retrievers, etc) configured in your local environment.
+
+Key features:
+
+- **Action Runners:** Sandboxed environments that let developers run Genkit flows and perform other actions like chatting with models, running structured prompts or testing retrievers.
```

Comment:
I don't agree with this term either. There's no sandboxing/isolation happening. We even don't really like calling it "emulator" because we're not emulating anything. The only thing is that we're emulating observability, so it's "isolated" in that sense, but a stretch IMO. @chrisraygill